tests/integration/components/bs-dropdown-test.js

```diff
@@ -365,6 +365,26 @@ module('Integration | Component | bs-dropdown', function (hooks) {
     assert.notStrictEqual(this.element.querySelector('.dropdown-menu').offsetParent, null);
   });
 
+  test('dropdown in nav disables dynamic positioning', async function (assert) {
+    await render(hbs`
+    <BsDropdown @inNav={{true}} as |dd|>
+      <dd.toggle>Dropdown</dd.toggle>
+      <dd.menu as |menu|>
+        <menu.item>
+          <menu.link-to @route="index">Home</menu.link-to>
+        </menu.item>
+      </dd.menu>
+    </BsDropdown>`);
+
+    await click('a.dropdown-toggle');
+
+    assert.equal(
+      this.element.querySelector('.dropdown-menu').style.length,
+      0,
+      'Dynamic positioning not applied when inNav is set to true'
+    );
```

Comment:
Do we have any test, which asserts the opposite? I mean that the element has custom styles if `@inNav` is not set (or set to `false`)? I fear otherwise this test can quickly turn to a false positive.

Does `@inNav` tracks changes? If so, I would prefer testing the assumption that it `style.length` is greater than `0` in the same test.

Reply:
I added before/after to this test. It initially starts with `inNav=false`, checks that the dynamic Popper styles are applied, flips `inNav=true` and checks that the dynamic styles are not applied. This should avoid a false positive in the future.